lib/editor/actions/trip.js

```diff
@@ -180,7 +180,10 @@ export function saveTripsForCalendar (
         })
     }))
       .then(trips => {
-        dispatch(fetchTripsForCalendar(feedId, pattern, calendarId, true))
+        // if error in saving, state will not refresh so user does not lose work
+        if (!trips.includes(undefined)) {
+          dispatch(fetchTripsForCalendar(feedId, pattern, calendarId, true))
+        }
```

Comment:
Same here -- could we possibly add an `else` and a console log saying that we haven't updated because we have an invalid trip!